Product.cs

```diff
@@ -10,24 +11,28 @@ public class Product
     {
         public string Name { get; set; }
         public int UPC { get; set; }
-        public decimal BasicPrice { get; set; }
+        public static decimal BasicPrice { get; set; }
 
```

Comment:
why do you define the product price as a static member? do you think it's a class variable? or it's specific to each object?